README.md

```diff
@@ -289,6 +289,19 @@ List of events:
 | entityCreated | Fired when a networked entity is created | `evt.detail.el` - new entity |
 | entityDeleted | Fired when a networked entity is deleted | `evt.detail.networkId` - networkId of deleted entity |
 
+The following events are fired on the `networked` component. See the [toggle-ownership component](./server/static/js/toggle-ownership.component.js) for examples.
+
+List of ownership transfer events:
+
+| Event | Description | Values |
+| -------- | ----------- | ------------- |
+| ownership-gained | Fired when a networked entity's ownership is taken | `evt.detail.el` - the entity whose ownership was gained |
+| | | `evt.detail.previousOwner` - the clientId of the previous owner |
+| ownership-lost | Fired when a networked entity's ownership is lost | `evt.detail.el` - the entity whose ownership was lost |
+| | | `evt.detail.newOwner` - the clientId of the new owner |
+| ownership-changed | Fired when a networked entity's ownership is changed, but you are neither the new or previous owner | `evt.detail.el` - the entity whose ownership was lost |
```

Comment:
Hmm I would have expected -`ownership-changed` to have fired any time the owner changes (including if it was me losing or becoming the owner). You could then actually get rid of the other events. Most of the times you use this you are going to have to handle all the cases, seems like it would be easier to just have a single event that forces you to do so.

Reply:
I considered that, but it seems like most aframe packages, (including aframe itself) tends to favor explicit events, as opposed to more generic events that have a `type` field in the evt.details. 
e.g. `child-attached`, `child-detached`; `stateadded`, `state-removed`

Reply:
Ah I see what you're suggesting down below. Though I disagree with "Most of the times you use this you are going to have to handle all the cases". I think by far the most used case will be the ownership-lost event, since that happens asynchronously, as opposed to taking ownership which is handled synchronously with whatever code is running that calls `takeOwnership()`. And in your example you always have to write `if(e.oldOwner === NAF.clientId)` to handle that event. I can see the case for both though.